util/ci/check_png_optimized.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+optimized=0
+
+for file in $(git ls-files *.png); do
+	 # Does only run a fuzzy check without -o7 -zm1-9 since it would be too slow otherwise
+	output=$(optipng -nc -strip all -simulate $file 2>&1 >/dev/null)
```

Comment:
1. $file should be quoted
2. did you check that it works? you redirect all output to dev null, how could this possibly ever capture anything?

Reply:
1. Changed
2. Yes it works. For some reason, optipng uses stderr for its output message, `2>&1 >/dev/null` redirects stderr to stdout and what would normally be in stdout to /dev/null.
e.g. try `(echo "in"; echo "err" >/dev/stderr) 2>&1 >/dev/null`